core/modules/installer/installer.manager.inc

```diff
@@ -1161,10 +1159,12 @@ function installer_manager_batch_project_get($project, $url, &$context) {
  */
 function installer_manager_local_transfers_allowed() {
   // Compare the owner of a webserver-created temporary file to the owner of
-  // the configuration directory to determine if local transfers will be
-  // allowed.
+  // the settings file to determine if local transfers will be
+  // allowed. The conf_path() may be owned by a different user but writable.
   $temporary_file = backdrop_tempnam('temporary://', 'update_');
-  $local_transfers_allowed = fileowner($temporary_file) === fileowner(conf_path());
+  $local_transfers_allowed = 
+    (fileowner($temporary_file) === fileowner(conf_path() . "/settings.php"))
+     && is_writable(conf_path());
```

Comment:
I think just putting it all on one line would make the linter happy.

```suggestion
  $local_transfers_allowed = (fileowner($temporary_file) === fileowner(conf_path() . "/settings.php")) && is_writable(conf_path());
```

Thanks for your patience.

Reply:
Thanks @quicksketch - I'm fairly sure that's what I had originally and it complained about line length.  Let's see!